packages/dx-react-bootstrap4/components/popover.jsx

```diff
@@ -84,7 +92,7 @@ export class Popover extends React.PureComponent {
             <div className="popover-inner" ref={this.contentRef}>
               {children}
             </div>
-            <div className="arrow" ref={arrowProps.ref} style={arrowProps.style} />
+            <ArrowComponent arrowProps={arrowProps} placement={restProps.placement} />
```

Comment:
I don't like `arrowProps` name

Reply:
Hm, I have no idea how to name `arrowProps` property. Maybe @MaximKudriavtsev and @ushkal can prompt?

Reply:
I think we can use `arrowProps` like in react-popper itself: https://github.com/FezVrasta/react-popper#children. It will be easier for user to understand this code

Reply:
`Popper.children.arrowProps` vs `DefaultArrowComponent. arrowProps`.
Does `ArrowComponent` have other props than `arrowProps`?